components/speeddial/index.tsx

```diff
@@ -0,0 +1,52 @@
+import { StyledSpeedDial } from './StyledSpeedDial'
+import {
+    SpeedDial as MuiSpeedDial,
+    SpeedDialIcon,
+    SpeedDialAction
+} from '@mui/material'
+import { RateReview, Message } from '@mui/icons-material'
+
+import { showDrawer } from '@/lib/redux/slices/drawer'
+import { useDispatch } from 'react-redux'
+import { SpeedDialProps } from '@/utils/types/components/speed-dial'
+import { DRAWER_TYPES } from '@/utils/constants'
+
+const { TESTIMONIAL, CLIENT_REQUEST } = DRAWER_TYPES
+const SpeedDialOptions: SpeedDialProps[] = [
+    { icon: <Message />, label: 'Contact Me', drawerDisplay: CLIENT_REQUEST },
+    {
+        icon: <RateReview />,
+        label: 'Create a Testimonial',
+        drawerDisplay: TESTIMONIAL
+    }
+]
+const SpeedDial = () => {
+    const dispatch = useDispatch()
+
+    const drawerToggle = (drawerDisplay: string) => {
```

Comment:
Lets get cute while we're in here. No need for the `{}` rapping the function. And i gave you the name updates
```
const onDrawerToggle = (content: string) => dispatch(showDrawer(content))
```